R/spinMyR.R

```diff
@@ -213,8 +215,14 @@ spinMyR <- function(file, wd, outDir, figDir, outSuffix,
               fileMd,
               quiet = !verbose,
               envir = spinMyR_Env)
+  if (!keepRmd) {
+    unlink(fileRmd)
```

Comment:
I'm not sure I need `recursive`, and why would we mute the warning when we can be reasonably sure that the file exists (otherwise `knit` would have thrown an error already)?